src/Utilities/OperatingSystemFilterHelpers.test.js

```diff
@@ -118,82 +64,82 @@ describe('groupOSFilterVersions', () => {
     expect(groupOSFilterVersions([])).toEqual([]);
   });
 
-  it('returns group configuration with 3 groups and 4 values', () => {
-    const config = groupOSFilterVersions(testVersions);
+  it('returns group configuration with 5 groups', () => {
+    const config = groupOSFilterVersions(availableVersions);
     expect(config).toMatchSnapshot();
     expect(config[0].groupSelectable).toEqual(true);
     expect(config[0].label).toEqual('RHEL 8');
-    expect(config[0].value).toEqual('8');
+    expect(config[0].value).toEqual('RHEL 8');
   });
 });
 
 describe('buildOSFilterChip', () => {
-  it('returns two version chips', () => {
-    const chips = buildOSFilterChip(testValue, testVersions);
+  it('returns five version chips', () => {
+    const chips = buildOSFilterChip(testValue2, availableVersions);
     expect(chips).toMatchSnapshot();
-    expect(chips[0].chips).toHaveLength(3);
+    expect(chips[0].chips).toHaveLength(5);
   });
 });
 
 describe('onOSFilterChange', () => {
   const clickedGroup = {
-    value: '8',
+    value: 'RHEL 8',
   };
   const clickedItem = {
-    value: '8.5',
+    value: '8.10',
   };
 
   it('returns new selection with group selection enabled', () => {
-    const selection = cloneDeep(testValue);
-    set(selection, [8, 8], false);
+    const selection = cloneDeep(testValue3);
+    selection['RHEL 8'] = {
+      // eslint-disable-next-line prettier/prettier
```

Comment:
This is not really necessary, or?

Reply:
are you referring to the eslint-disable? It is necessary. It doesn't like `'8.5'`. It wants me to put `8.5`, but when I do that, it messes up the `toEqual`.

Reply:
ignore my last comment